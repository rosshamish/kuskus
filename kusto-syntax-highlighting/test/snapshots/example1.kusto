StormEvents 
| where isnotempty(EndLocation) 
| summarize event_count=count() by EndLocation
| top 10 by event_count
| render columnchart
