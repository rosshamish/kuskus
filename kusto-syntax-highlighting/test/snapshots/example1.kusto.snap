>StormEvents 
#^^^^^^^^^^^^^ source.kusto
>| where isnotempty(EndLocation) 
#^^ source.kusto
#  ^^^^^ source.kusto keyword.other.query.kusto
#       ^ source.kusto
#        ^^^^^^^^^^ source.kusto support.function.kusto
#                  ^^^^^^^^^^^^^^^ source.kusto
>| summarize event_count=count() by EndLocation
#^^ source.kusto
#  ^^^^^^^^^ source.kusto keyword.other.query.kusto
#           ^^^^^^^^^^^^^ source.kusto
#                        ^^^^^^^ source.kusto support.function.kusto
#                               ^ source.kusto
#                                ^^ source.kusto keyword.other.operator.kusto
#                                  ^^^^^^^^^^^^^ source.kusto
>| top 10 by event_count
#^^ source.kusto
#  ^^^ source.kusto meta.query.top.kusto keyword.other.query.kusto
#     ^ source.kusto meta.query.top.kusto
#      ^^ source.kusto meta.query.top.kusto constant.numeric.kusto
#        ^ source.kusto meta.query.top.kusto
#         ^^ source.kusto meta.query.top.kusto keyword.other.operator.kusto
#           ^ source.kusto meta.query.top.kusto
#            ^^^^^^^^^^^ source.kusto meta.query.top.kusto variable.other.column.kusto
>| render columnchart
#^^ source.kusto
#  ^^^^^^ source.kusto keyword.other.query.kusto
#        ^ source.kusto
#         ^^^^^^^^^^^ source.kusto support.function.kusto
>